### ISSUE TITLE
Cannot cast Price of Fame if Mana open is less than 4.

### ISSUE THREAD

**Issue by rexelallan** [magefree/mage#6685]
As the title said.

I have 3 mana open, I cannot cast Price of Fame.

When I flicker my land through Aminatou, It enables me to cast it since I have now 4 lands open.

**Comment by SergeWind** [comment]
Same issue here, but with Wizard's Lighting and 1 mountain + Monastery Swiftspear in play; some of my friends are also saying they can't cast the new Of One Mind without 3 untapped lands either. I wonder if it's all connected, and related to a new update, hmmm.

**Comment by Zerrisx** [comment]
Duplicate of #6698

**Comment by Zerrisx** [comment]
Looks like this was fixed.

**Comment by JayDi85** [comment]
Nope, it's still bugged and can't cast without full price available:
```
[test Price of Fame reduction]
hand:Human:Price of Fame:1
battlefield:Human:Swamp:2
battlefield:Computer:Anje Falkenrath:1
```

I'm working on it.

``` java
    @Test
    public void test_PriceOfFame_Reduce() {
        // {3}{B}
        // This spell costs {2} less to cast if it targets a legendary creature.
        // Destroy target creature.
        addCard(Zone.HAND, playerA, "Price of Fame", 1);
        addCard(Zone.BATTLEFIELD, playerA, "Swamp", 4 - 2);
        addCard(Zone.BATTLEFIELD, playerB, "Anje Falkenrath", 1);

        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Price of Fame", "Anje Falkenrath");

        setStrictChooseMode(true);
        setStopAt(1, PhaseStep.BEGIN_COMBAT);
        execute();
        assertAllCommandsUsed();

        assertGraveyardCount(playerB, "Anje Falkenrath", 1);
    }
```

**Comment by Zerrisx** [comment]
Whoops - I had linked Of One Mind and Wizard's Lightning to here, both of which were fixed, but you're right; the original Price of Fame is not.

**Comment by JayDi85** [comment]
Fixed via 520d75dba9cddc814a737dd5670257e3a2f9cd57
